### PR TITLE
docs: add table for unsupported types in Hive JDBC driver

### PR DESCRIPTION
**Pull request by bradfordb-vmware**
Authored-by: Bradford D. Boyle <bradfordb@vmware.com>

### CODE REVIEW

**Comment by mperezfuster** [comment]
LGTM, just a couple of minor changes.

File: docs/content/jdbc_pxf.html.md.erb [hunk 1]
```diff
@@ -53,7 +53,7 @@ The PXF JDBC connector supports the following data types:
 
 Any data type not listed above is not supported by the PXF JDBC connector.
 
-**Note**: PXF includes Hive JDBC driver 1.1.0 which does not support the following data types:
+**Note**: PXF includes Hive JDBC driver version 1.1.0 which does **not** support the following data types when reading from or writing to Hive tables.
```
Comment (bradfordb-vmware) [inline docs/content/jdbc_pxf.html.md.erb:56]: I double checked and with the exception of `binary`/`bytea`, this table is only for _writing_ to Hive. Should we add another column for whether it is read or write (or both)?

**Comment by lisakowen** [comment]
@bradfordb-vmware @mperezfuster - can you check out my recent commit to this PR and let me know what you think?

**Comment by lisakowen** [comment]
merging this PR.

**Pull request merged**